src/main/java/com/jakub/todoSandbox/model/Todo.java

```diff
@@ -0,0 +1,47 @@
+package com.jakub.todoSandbox.model;
+
+import java.util.List;
+
+public record Todo (Long id, String name, String description, Priority priority, List<Step> steps){
+
+
+    public static Builder builder(String name) {
```

Comment:
Why this is needed?

Reply:
@BinaryIgor I created it in order to make the code generation look cleaner in the JOOQRepository class. Should I delete it?